content/index.de.md

```diff
@@ -138,17 +141,17 @@ Barrierefreiheit ist in vielen Fällen **gesetzlich verpflichtend**.
 
 ## Das Web barrierefrei machen {#making}
 
-Barrierefreiheit im Web hängt vom Zusammenspiel zahlreicher Komponenten ab, unter anderem Webtechnologien, Webbrowser und anderen \"User Agents\", Erstellungswerkzeugen und Webseiten.
+Barrierefreiheit im Web hängt vom Zusammenspiel zahlreicher Komponenten ab, unter anderem Webtechnologien, Webbrowsern und anderen \"User Agents\", Erstellungswerkzeugen und Webseiten.
 
-Die W3C Initiative für barrierefreies Web, Web Accessibility Initiative ([WAI](/about/participating/)) entwickelt technische Spezifikationen, Richtlinien, Techniken sowie unterstützende Ressourcen, die Lösungen für Barrierefreiheit beschreiben. Diese gelten als internationale Standards für barrierefreies Web; die Richtlinien für barrierefreie Webinhalte, <abbr title="Web Content Accessibility Guidelines (WCAG)">WCAG</abbr> 2.0 ist beispielsweise auch ein <abbr title="International Organization for Standardization">ISO</abbr> Standard: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.
+Die W3C-Initiative für barrierefreies Web – die Web Accessibility Initiative ([WAI](/about/participating/)) – entwickelt technische Spezifikationen, Richtlinien, Techniken sowie unterstützende Ressourcen, die Lösungen für Barrierefreiheit beschreiben. Diese gelten als internationale Standards für barrierefreies Web. Die Richtlinien für barrierefreie Webinhalte, <abbr title="Web Content Accessibility Guidelines (WCAG)">WCAG</abbr> 2.0, ist beispielsweise auch ein <abbr title="International Organization for Standardization">ISO</abbr> Standard: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.
 
 {::nomarkdown}
 {% include box.html type="start" h="3" title="Weitere Infos zum barrierefreien Gestalten des Webs" class="simple aside" %}
 {:/}
 
 -   Mehr zum Zusammenspiel dieser Aspekte gibt es unter [[Wesentliche Komponenten des barrierefreien Webs]](/fundamentals/components/).
 -   Die Richtlinien für barrierefreie Webinhalte (WCAG), die Richtlinien für barrierefreie Erstellungswerkzeuge (ATAG), den Standard für barrierefreie Internetanwendungen (ARIA), und weitere wichtige Ressourcen werden eingeführt unter [[Übersicht über die Standards für Barrierefreiheit des W3C]](/standards-guidelines/).
--   Mehr darüber wie W3C WAI durch die internationale Beteiligung vieler Interessensvertreter:innen Material entwickelt und wie Sie beitragen können, finden Sie in [[Über WAI]](/about/) und unter [[Sich bei WAI beteiligen]](/about/participating/).
+-   Mehr darüber wie W3C-WAI durch die internationale Beteiligung vieler Interessensvertreter:innen Material entwickelt und wie Sie beitragen können, finden Sie in [[Über WAI]](/about/) und unter [[Sich bei WAI beteiligen]](/about/participating/).
```

Comment:
"W3C WAI" is indeed presented without a dash or hyphen.

Reply:
I’m not sure how to read this comment—what do you mean by “indeed”?

In German, nouns are connected, that is, either “merged” as one word, or joined by a hyphen. (Another correct form would be to use a possessive, like “WAIs WAI,” but that would be different from the English version.)

This is handled less and less strict because of the English influence on German (my read), so I could change it back, but again, I’m not sure how to read the comment.

Reply:
(Will for now remove the dashes and defer to @alexejrotar to eventually readjust.)